rename r to R-arrow 2

Tmonster opened the pull request.

Tmonster committed:
should change arrow to show R-arrow

Tmonster committed:
new arrow benchmarks report solution as R-arrow

Tmonster committed:
update arrow to R-arrow in a few more places

amoeba commented:
From the CI failure it looks like there's one more spot to tweak, I'll see if I can track it down.

amoeba commented:
I've put up a PR at https://github.com/Tmonster/db-benchmark/pull/13 which does the remainder of the renaming and makes CI pass.

amoeba committed:
Fix remaining issues in https://github.com/Tmonster/db-benchmark/pull/10 (#13)

* Fix remaining issues in arrow -> R-arrow rename

* Fix bug in rename code in report.R

The previous code was causing something wild to happen. The changed code is idiomatic code for replacing values in a data.frame based on a condition.

amoeba commented:
My pr of the same name was based off the tip of your PR so merging my branch into your and then merging that to master should do it. Thanks!
